src/main/java/com/puppycrawl/tools/checkstyle/checks/design/FinalClassCheck.java

```diff
@@ -40,9 +40,28 @@
 
 /**
  * <p>
- * Checks that a class that has only private constructors and has no descendant
- * classes is declared as final.
+ * Check that identifies classes that can be declared as final are explicitly made final.
+ * The following is the type of classes that can be identified:
```

Comment:
```suggestion
 * Ensures that classes that can be effectively declared as final are explicitly marked as final. The following are 
 * different types of classes that can be identified:
```

Reply:
@nrmancuso Done

Reply:
Btw, @Kevin222004 I did `improve: 'Check that identifies classes that can be declared as final are explicitly made final. The following is the type of classes that can be identified:'`  in chatGPT to generate this, do not be afraid to do the same ;) We have crazy new AI tools out there, use them to your advantage :)